NeoML/docs/en/API/NN/MathEngine.md

```diff
@@ -140,14 +140,13 @@ This math engine should be deleted after use.
 ### Create a CPU math engine
 
 ```c++
-IMathEngine* CreateCpuMathEngine( int threadCount, size_t memoryLimit );
+IMathEngine* CreateCpuMathEngine( size_t memoryLimit );
 ```
 
 Creates a math engine working on CPU, setting the memory limitation, the number of threads and the custom exception handler.
```

Comment:
Update comment plz. Same goes for Russin docs.

Reply:
fixed